doc/classes/Engine.xml

```diff
@@ -51,6 +51,13 @@
 				Returns the frames per second of the running game.
 			</description>
 		</method>
+		<method name="get_idle_frames" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the total number of frames passed since engine initialization which is advanced on each [b]idle frame[/b], regardless of whether render loop is enabled, see [method get_frames_drawn].
```

Comment:
"whether **the** render loop"
I'd also do `s/, see/. See also/`.

Reply:
Replaced with `See also`. Not sure what you mean by `/`.

Reply:
That was `sed` syntax for a substitution :)
`s/a/b/` means "replace 'a' by 'b'".